subsys/spm/Kconfig

```diff
@@ -0,0 +1,87 @@
+#
```

Comment:
I am not sure this is the right approach.
The Secure Partition Manager is supposed to be a generic subsystem.
CONFIG_SPM shall bring-in the core of the Secure Partition Manager, e.g. in theory, this code should be platform-agnostic.

In theory, this could be upstreamed, at some point.

I wonder if we could do something like Bluetooth, where there is a "nordic" folder inside the controller/ll_sw and all the implementation in the controller level is not Nordic-specific.

Reply:
All configuration in this file is nordic specific. The entire `nrf/subsys/spm` folder should be located in   `zephyr/subsys/spm/nordic/` when being upstreamed. 

IMO we should leave it as is for now, since adding another layer of folders does not give anything of value AFAICS.

Reply:
Alright.
This Kconfig file is 100% Nordic-specific, right? I wonder if it would be better to rename this to Kconfig.nrf; that's what we have for drivers' Kconfig files

Reply:
Isn't the "Kconfig.xxx" used in zephyr to avoid  recursive inclusion?

Reply:
Not sure. But in any case, this would be needed if we want to upstream an SPM sub-system...